crates/ibc/src/mock/header.rs

```diff
@@ -70,15 +70,17 @@ impl MockHeader {
     }
 
     pub fn new(height: Height) -> Self {
-        if cfg!(any(test, feature = "std")) {
-            Self {
-                height,
-                timestamp: Timestamp::now(),
-            }
-        } else {
-            Self {
-                height,
-                timestamp: Timestamp::none(),
+        cfg_if::cfg_if! {
+            if #[cfg(any(test, feature = "std"))] {
+                Self {
+                    height,
+                    timestamp: Timestamp::now(),
+                }
+            } else {
+                Self {
+                    height,
+                    timestamp: Timestamp::none(),
+                }
```

Comment:
IMO we can make `MockHeader::new` return with `Timestamp::none()`. and add another method `with_current_timestamp()` with `#[cfg(any(test, feature = "std"))]`.

Reply:
btw, do we need `cfg_if`? [we can just do](https://doc.rust-lang.org/rust-by-example/attribute/cfg.html)
```
if cfg!(any(test, feature = "std)) {
    ...
} else { 
    ...
}
```

Reply:
`cfg_if` was dropped since mock types moved to the `ibc-testkit` and `ibc-testkit` doesn't well support `no-std`